en/flight_controller/beaglebone_blue.md

```diff
@@ -65,102 +58,88 @@ Currently running on a:
 MODEL_BB_BLUE
 Robot Control library Version:
 1.0.0
-
 ```
 
-As of this writing, the latest OS images in which librobotcontrol works properly
-is [bone-debian-9.5-iot-armhf-2018-07-22-4gb.img.xz](https://rcn-ee.net/rootfs/bb.org/testing/2018-07-22/stretch-iot/bone-debian-9.5-iot-armhf-2018-07-22-4gb.img.xz).
+> **Tip** Optionally you can update to a realtime kernel, and if you do, re-check if *librobotcontrol* works properly with the realtime kernel.
 
-#### Setup Robot Control Library
```

Comment:
@UAV-Pilot it isn't clear which parts need to be done on dev computer vs Beaglebone Blue or both.

Reply:
If user chooses native build, then he/she does not need setup a separate dev computer, so librobotcontrol setup always need be setup in Beaglebone Blue as the executable need load the library at run time, and it only need be setup on the dev computer if the user chooses cross compile.